nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -62,7 +64,9 @@ def build_url_for_details(repo_details)
           body = remove_wrapping_zero_width_chars(response.body)
           base_url = base_url_from_v3_metadata(JSON.parse(body))
           resolved_base_url = base_url || repo_details.fetch(:url).gsub("/index.json", "-flatcontainer")
-          search_url = search_url_from_v3_metadata(JSON.parse(body))
+          parsed_json = JSON.parse(body)
```

Comment:
We should move this up so we can use it when getting the base_url from the response body.